general/g.copy/g.copy.html

```diff
@@ -1,13 +1,48 @@
 <h2>DESCRIPTION</h2>
 
+The <em>g.copy</em> module creates a copy of existing raster maps,
+vector maps, or other elements. The copy is always created in
+the current mapset. The source data can be in the current mapset,
+in an explicitly specified mapset, or in a mapset which is in the
+current mapset search path (typically the PERMANENT mapset).
+
+<p>
+The maps and other elements to copy are specified in pairs
+<b>from</b>,<b>to</b> according to their types.
+Although typically only one map is copied in one module call,
+multiple pairs can be provided for each type and multiple types
+can be provided at the same time.
+
+<h3>Relation to mapsets</h3>
+
 A user may access data stored under the other mapsets listed in their
 mapset search path. However, the user may only modify data stored
 under their own current mapset. <em>g.copy</em> allows the user to copy
 existing data files <b>from</b> other mapsets <b>to</b> the user's
-current mapset ($MAPSET). The files to be copied must exist in the
+current mapset (<code>g.mapset -p</code>). The files to be copied must exist in the
 user's current mapset search path and location; output is sent to the
 relevant data element directory(ies) under the user's current mapset.
 
+<h3>Behavior on error</h3>
+
+When only one map or other element is requested to be copied and the copying
+is not possible or fails, an error is reported.
+
+<p>
+Multiple maps or other elements can be copied using the same command.
+In that case, <em>g.copy</em> attempts to copy as much as possible
+even when problems occur with one of the elements.
+If nothing can be copied or an error occurred during one of the copy
+operations, an error message is reported.
+Copying is skipped when a map or other element does not exist,
+<b>from</b> and <b>to</b> are the same, <b>to</b> element already exists
+and overwriting (e.g., by <b>--overwrite</b>) is not enabled
+or <b>to</b> element has an illegal name.
+Copying of remaining elements is not prevented by skipping or copying error
+of another element.
+However, as stated above, module ends with an error when copying failed,
+but only after other copy operations were performed.
```

Comment:
```suggestion
In that case, copying of the element causing problems is skipped, and
<em>g.copy</em> proceeds with copying the  remaining elements.

If nothing can be copied or an error occurred during one of the copy
operations, an error message is reported  after other possible copy
operations were performed.
```
Moved one sentence up, dedulicate language a little.